### PR TITLE
Implement makeValid

### PR DESCRIPTION
**Pull request by mgraber**
#17

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ boto3 = "^1.16.58"
 python-dotenv = "^0.15.0"
 SQLAlchemy = "^1.3.22"
 psycopg2-binary = "^2.8.6"
-GDAL = "*"
+GDAL = "^3.2.1"
```
Comment (SPTKL) [inline pyproject.toml:17]: why are we setting explicit version here?

Comment (mgraber) [reply pyproject.toml:17]: I thought we needed to for MakeValid to work.

Comment (SPTKL) [reply pyproject.toml:17]: we don't know when makevalid was incorporated to GDAL

Comment (SPTKL) [reply pyproject.toml:17]: but yeah, it's probably for the best

**Pull request merged**